src/views/Home.vue

```diff
@@ -1,46 +1,104 @@
-          <ion-title size="large">{{ $t('Blank') }}</ion-title>
-        </ion-toolbar>
-      </ion-header>
-    
-      <div id="container">
-        <strong>{{ $t('Ready to create an app?') }}</strong>
-        <p>{{ $t('Start with Ionic') }} <a target="_blank" rel="noopener noreferrer" href="https://ionicframework.com/docs/components">{{ $t('UI Components') }}</a></p>
-      </div>
+      <ion-text id="title">
+        <h1>{{ $t("Your Order") }}</h1>
+      </ion-text>
+      <ion-card>
+        <ion-item lines="none">
+          <ion-label>
+            <h2>Customer Name</h2>
+            <p>Order ID</p>
+          </ion-label>
+          <ion-note slot="end">order date</ion-note>        
+        </ion-item>
+        <ion-item lines="full">
+          <ion-thumbnail slot="start">
+            <Image src="https://gravatar.com/avatar/dba6bae8c566f9d4041fb9cd9ada7741?d=identicon&f=y"></Image>
+          </ion-thumbnail>
+          <ion-label slot="start">
+            <p>BRAND</p>
+            <h2>Virtual Name</h2>
+            <p>{{ $t("Color") }}: color</p>
+            <p>{{ $t("Size") }}: size</p>
+          </ion-label>
+          <ion-badge slot="end" color="primary">status</ion-badge>
+        </ion-item>
+        <ion-item>
+          <ion-label>{{ $t("Store pickup") }}</ion-label>
+          <ion-button @click="openPickupPopover"  color="medium" fill="outline">Change</ion-button>
+        </ion-item>
+        <ion-item lines="full">
+          <ion-label>{{ $t("Ship to") }}</ion-label>  
+          <ion-button  color="medium" fill="outline" @click="openShipmentModal">Edit</ion-button>
+        </ion-item>
+        <ion-item lines="none">
+          <ion-label color="danger">{{ $t("Cancel Item") }}</ion-label>  
+        </ion-item>
+      </ion-card>
     </ion-content>
   </ion-page>
 </template>
 
 <script lang="ts">
-import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/vue';
-import { defineComponent } from 'vue';
+import {
+  IonBadge,
+  IonButton,
+  IonCard,
+  IonContent,
+  IonItem,
+  IonLabel,
+  IonNote,
+  IonPage,
+  IonText,
+  IonThumbnail,
+  popoverController,
+  modalController  
+} from "@ionic/vue";
+import { defineComponent } from "vue";
+import Popover from "./PickupPopover.vue";
+import Image from "@/components/Image.vue";
+import ShipmentModal from "./ShipmentModal.vue";
 
 export default defineComponent({
-  name: 'Home',
   components: {
-    IonContent,
-    IonHeader,
-    IonPage,
-    IonTitle,
-    IonToolbar
-  }
+   Image,
+   IonBadge,
+   IonButton,
+   IonCard,
+   IonContent,
+   IonItem,
+   IonLabel,
+   IonNote,
+   IonPage,
+   IonText,
+   IonThumbnail,
+  },
+  methods: {
+    async openPickupPopover(ev: Event) {
+      const popover = await popoverController.create({
+        component: Popover,
+        cssClass: "my-custom-class",
+        event: ev,
+        translucent: true,
+      });
+      await popover.present();
+    },
+
+    async openShipmentModal() {
```

Comment:
Improve method name based upon it's purpose. Something like updateShipmentAddress